science/OpenCoarrays/Portfile

```diff
@@ -31,6 +31,13 @@ checksums           rmd160  3e94a9fe478d60e027730e596ffe1c384de5da77 \
                     sha256  c08717aea6ed5c68057f80957188a621b9862ad0e1460470e7ec82cdd84ae798 \
                     size    329486
 
+pre-patch   {
+    file rename ${worksrcpath}/src/tests/integration/pde_solvers/coarrayHeatSimplified/main.f90 \
+        ${worksrcpath}/src/tests/integration/pde_solvers/coarrayHeatSimplified/main.F90_
+    file rename ${worksrcpath}/src/tests/integration/pde_solvers/coarrayHeatSimplified/main.F90_ \
+        ${worksrcpath}/src/tests/integration/pde_solvers/coarrayHeatSimplified/main.F90
+}
```

Comment:
Ah, I didn't realize you were using `file rename`.

Try using `move` instead, which should eliminate the need for a two-step process:

https://guide.macports.org/#reference.tcl-extensions

Reply:
@mascguy Gonna try that and update today in a couple of hours.